eras/conway/test-suite/cddl-files/conway.cddl

```diff
@@ -267,12 +269,16 @@ delegatee =
   // 2, pool_keyhash, voting_credential
   ]
 
+
+noop_cert = (0, stake_credential)
+stake_deregistration = (1, stake_credential)
+stake_delegation = (2, stake_credential, pool_keyhash)
 pool_registration = (3, pool_params)
 pool_retirement = (4, pool_keyhash, epoch)
 constitutional_deleg_cert = (5, genesishash, genesis_delegate_hash, vrf_keyhash)
```

Comment:
what happened to `6`? why did these get bumped?

Reply:
`6` used to be the MIR certificates. In Conway trying to decode a MIR cert will give an error, but I'm not sure if we actually need to keep that around.

Reply:
ah yes, thanks! maybe we could leave a comment saying that `6` was MIR, which is now deprecated?